pegasus/sites.v3/code.org/public/educate/resources/videos.md

```diff
@@ -37,6 +37,14 @@ We are building a growing library of educational videos available for re-use by
 
 </div>
 
+<div style="float:left; padding:10px; width:49%">
+
+<% facebook = {:u=>'https://youtu.be/Dxcc6ycZ73M'} %>
```

Comment:
I prefer the shorter URL if possible (since that's the share link), but longer version if needed like when we have the start watch time.